tests/BenchmarkDotNet.IntegrationTests/StringExtensions.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace BenchmarkDotNet.IntegrationTests;
+
+internal static class StringExtensions
+{
+    private static readonly char[] newLine = Environment.NewLine.ToCharArray();
+
+    /// <summary>
+    /// Creates an array of strings by splitting this string at each occurrence of a newLine separator.
+    /// Contrary to calling <code>value.Split('\r', '\n')</code>, this method does not return an empty
+    /// string when CR is followed by LF.
+    /// </summary>
+    public static IEnumerable<string> EnumerateLines(this string value)
```

Comment:
Please use `StringReader` and its [ReadLine](https://learn.microsoft.com/en-us/dotnet/api/system.io.stringreader.readline?view=net-9.0) method instead.